core/meta/inc/TInterpreter.h

```diff
@@ -559,9 +559,6 @@ class TInterpreter : public TNamed {
    virtual Bool_t IsPointerType(const void * /* QualTypePtr */) const {return 0;}
    virtual Bool_t IsVoidPointerType(const void * /* QualTypePtr */) const {return 0;}
 
-   // FunctionDecl interface
-   virtual Bool_t FunctionDeclId_IsMethod(DeclId_t /* fdeclid */) const {return 0;}
```

Comment:
This is in the public.  Technically (but unlikely) this might be used by some users.

Reply:
You're right, but given that it was only introduced 2 years ago for the sake of cppyy, I doubt that. We still have the opportunity to remove this public interface again I'd say, and slimming the interface is always good